PROTOCOL.md

```diff
@@ -1595,23 +1621,24 @@ Reader Version 3 | Respect [Table Features](#table-features) for readers<br> - W
 
 ## Valid Feature Names in Table Features
 
-Feature | Name | Readers or Writers?
--|-|-
-[Append-only Tables](#append-only-tables) | `appendOnly` | Writers only
-[Column Invariants](#column-invariants) | `invariants` | Writers only
-[`CHECK` constraints](#check-constraints) | `checkConstraints` | Writers only
-[Generated Columns](#generated-columns) | `generatedColumns` | Writers only
+Feature | Name                  | Readers or Writers?
```

Comment:
are all these whitespace changes needed? i think the existing table renders just fine, and having these whitespaces seems unnecessary.

Reply:
Intellij was complaining about `For compatibility reasons, all table rows should have borders (pipe symbols) at the start and at the end` but it renders fine.